packages/database/src/migrations/20210419225536-UpdatePSSSTotalSitesReportToUseSiteData-modifies-data.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+import { generateId, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const oldConfig = {
+  formula: 'PSSS_Sites_Reported',
+  aggregation: 'FINAL_EACH_WEEK',
+};
+
+const newConfig = {
+  formula: `firstExistingValue(PSSS_Sites_Reported, PSSS_Site_Weekly_Survey_Completed, PSSS_Site_Daily_Survey_Completed)`,
+  aggregation: {
+    PSSS_Sites_Reported: 'FINAL_EACH_WEEK',
+    PSSS_Site_Weekly_Survey_Completed: [
+      'FINAL_EACH_WEEK',
+      {
+        type: 'SUM_PER_PERIOD_PER_ORG_GROUP',
+        config: { dataSourceEntityType: 'facility', aggregationEntityType: 'country' },
+      },
+    ],
+    PSSS_Site_Daily_Survey_Completed: [
+      'FINAL_EACH_WEEK',
```

Comment:
Should this be `FINAL_EACH_DAY`? (sum data for all days in a week)

Reply:
Nope, since this aggregation will just select a single analytic per org unit per week which is what we want (multiple daily submissions from the same site in on week should only count as 1)

Reply:
Oh true, since this is for counting "how many submitted in a week" rather than summing any results... Thanks!